src/crypto/a128kw.rs

```diff
@@ -79,3 +118,18 @@ impl JweA128KWEncipher {
         })
     }
 }
+
+fn kid(wrap_key: &Aes128Key) -> String {
+    let mut skey = HmacSha256Key::default();
+    let skey_slice = skey.as_mut_slice();
+    let wrap_key_slice = wrap_key.as_slice();
+    let skey_slice_mut = &mut skey_slice[..wrap_key_slice.len()];
+    skey_slice_mut.copy_from_slice(wrap_key_slice);
+    // Key is setup
+    let mut hmac = HmacSha256::new(&skey);
```

Comment:
And then not using the updated thing at all?

Reply:
because skey_slice_mut is a view ito skey, once the cope_from_slice is done, it's good to go.